lib/smack/SmackRep.cpp

```diff
@@ -13,6 +15,11 @@ const string SmackRep::BOOL_TYPE = "bool";
 const string SmackRep::FLOAT_TYPE = "float";
 const string SmackRep::NULL_VAL = "$NULL";
 
+const string SmackRep::BYTE_TYPE = "i8";
```

Comment:
Probably should remove this, preferring the use of `int_type(8)` in place of `getByteType()`.